lib/client.js

```diff
@@ -41,6 +41,7 @@ var Client = function Client(dsn, options) {
   this.root = options.root || process.cwd();
   this.transport = options.transport || transports[this.dsn.protocol];
   this.release = options.release || process.env.SENTRY_RELEASE || '';
+  this.environment = options.environment || process.env.SENTRY_ENVIRONMENT || '';
```

Comment:
I kind of agree, and was going to ping @dcramer whether SENTRY_ENVIRONMENT was a thing.